.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+---
+
```

Comment:
Why this extra line?

Reply:
@devkapilbansal this was unusual, I will remove it. Also will remove the matrix portion too.